pyronear/datasets/openfire.py

```diff
@@ -112,7 +114,7 @@ def download(self, threads=None):
         # Download annotations
         download_url(self.url, self._root.joinpath(self._raw), filename=self.url.rpartition('/')[-1], verbose=False)
         with open(self._root.joinpath(self._raw, self.url.rpartition('/')[-1]), 'rb') as f:
-            annotations = json.load(f)
+            annotations = json.load(f)[:num_samples]
```

Comment:
I'm not sure, but shouldn't it be:
```
annotations = json.load(f)
if num_samples is not None:
    annotations = annotations[:num_samples]
```

Otherwise, slicing with None will result in error

Reply:
None is a perfectly valid argument for a slice and keeps all the list
```
In [4]: range(10)[:None]
Out[4]: range(0, 10)
```

Reply:
My bad then, thanks for your response!